en/02-elm-arch/03-messages.md

```diff
@@ -4,8 +4,10 @@
 
 In the last section, we created an application using Html.program that was just static Html. Now let's create an application that responds to user interaction using messages.
 
+Now, edit your **App.elm** file, or create a new one (just remember to give the module the same name that your source file has):
+
```

Comment:
In https://github.com/sporto/elm-tutorial-app/blob/master/src/Main.elm I named the app `Main`. 
I think this is great, but please change to `Main.elm`

Reply:
Don't mind this. You are right, this part of the tutorial uses `App.elm`. I might need to make this consistent later.